ebean-core/src/main/java/io/ebeaninternal/server/type/ScalarTypeJsonObjectMapper.java

```diff
@@ -66,6 +70,66 @@ public boolean isJsonMapper() {
     public String jsonMapper(Object value) {
       return formatValue(value);
     }
+    
+    private class Md5MutableJson implements MutableJson {
+
+      private String md5;
+      Md5MutableJson(String json) {
+        md5 = Md5.hash(json);
+      }
+      @Override
+      public boolean isEqualToObject(Object obj) {
+        return true; // we cannot determine differences...
+      }
+
+      @Override
+      public boolean isEqualToJson(String json) {
+        return Md5.hash(json).equals(md5);
+      }
+
+      @Override
+      public Object get() {
+        return null; // cannot create object from json
+      }
+      @Override
+      public void update(Object obj) {
+        md5 = Md5.hash(format(obj));
```

Comment:
I tried to make that class static, to have a lower memory footprint. But I need the "update" method.